var/spack/repos/builtin/packages/chameleon/package.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class Chameleon(CMakePackage):
+    """Dense Linear Algebra for Scalable Multi-core Architectures and GPGPUs"""
+    homepage = "https://gitlab.inria.fr/solverstack/chameleon"
+    git      = "https://gitlab.inria.fr/solverstack/chameleon.git"
+    maintainers = ['fpruvost']
+
+    version('1.1.0', 'e64d0438dfaf5effb3740e53f3ab017d12744b85a138b2ef702a81df559126df',
+            url='https://gitlab.inria.fr/solverstack/chameleon/uploads/b299d6037d7636c6be16108c89bc2aab/chameleon-1.1.0.tar.gz')
+    version('master', branch='master', submodules=True)
```

Comment:
This should stay on top of releases (master > 1.1.0 and usually we employ a descending order for version directives)